src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
-public class Application {
+public class Application extends Lottery {
```

Comment:
```suggestion
public class Application {
```
you don't need to extend Lottery here, you should create an instance of it and call its method